Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -48,6 +48,29 @@
   --chromium-border-radius: 1px;
   --mozilla-box-shadow: 0 0 0 2px royalblue;
   --mozilla-border-radius: 1px;
+  --remotion-themed-font-heading: 'Roboto-Bold', sans-serif;
+  --remotion-themed-font-bold: 'Roboto-Medium', sans-serif;
+  --remotion-themed-font-standard: 'Roboto-Regular', sans-serif;
+}
+
+@font-face {
+  font-family: 'Roboto-Medium';
+  src: url('../../../Html/Roboto-Medium.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Roboto-Bold';
+  src: url('../../../Html/Roboto-Bold.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Roboto-Regular';
+  src: url('../../../Html/Roboto-Regular.ttf') format('truetype');
+}
+
+*
+{
+  font-family: var(--remotion-themed-font-standard);
 }
```

Comment:
It is sad that we have to do it like this because now fonts are no longer inherited and need to be applied to every control. Maybe it would be better to apply it only to `body` and reset all the pre-defined browser styles that explicitly set a `font-family`, but this is a lot more work ^^

Reply:
What would be the benefit of doing it this way, because `*` already selects all elements on the site and changes the font.🤔

Reply:
If you want to change the font you have to child elements because of the `*` selector. So if you have a `<code>` block that should have another font and you want, for example, to use an `<b>` element inside your CSS needs to explicitly handle the `<b>` tag because the `*` rule overrides the font again.

I don't actually know if this is an important thing for us or if it can stay this way (cc @MichaelKetting)

Reply:
we don't want a * rule here. it may be valid, that we declare the css variables in Common.css and then apply the standard-font via style.css. This does not solve the question how we deal with the individual controls. I don't hink this can be solved via review-ping-pong?

Reply:
In `style.css` there is already a rule for the `font-family`.
```css
body, input, select, textarea, button
{
  font-family: 'Arial', 'Helvetica', sans-serif;
}
```
An easy solution would be to use the same selectors in the `common.css` rule.

Reply:
How was the issue with the controls handled in before?

Reply:
before, we just said that by importing style.css, we globally define the fonts. this decision (importing style.css) was done by the application developer, not by the controls library author. so, what I want is that just because we add common.css, this should not globally set the font. globally setting the font may only be done via style.css.

maybe the easiest way would be if we define the three font-variables in common.css without specifiying the font, setting the value of the font-variables in style.css and using the font-variables for bold and medium at those places where we need to use medium or bold font.

You can include the weight in the generic font, see here: https://stackoverflow.com/questions/4218549/one-css-declaration-for-all-css-font-properties
`font: bold italic 15px/20px arial,sans-serif;`

That way, we can make the medium and bold font variables to just specify the weight in common.css and then override via style.css. it's not pefect, but probably a good comproimise.